src/com/raghav/javabasics/Counter.java

```diff
@@ -0,0 +1,55 @@
+package com.raghav.javabasics;
+// words, char, lines
+
+public class Counter {
+    public static void main(String[] args) {
+      String paragraph = "Change an Array\n" +
+              "new\n" +
+              "out\n" +
+              "Element";
+      Counter cnt = new Counter();
+     int[] temp =  cnt.wordsCharLines(paragraph);
+        System.out.println(paragraph);
+
+        for (int i=0;i<temp.length;i++)
+        System.out.println(temp[i]);
+    }
+    public int words(String paragraph){
+        int wordCount=1;
+        for(int i=0;i<paragraph.length();i++){
+
+            char c = paragraph.charAt(i);
+            if(c == ' ' || c == '\n'){
```

Comment:
What happens if there are some special characters in the string? Add testcase for the same.
String paragraph=“this,is.also.a.sentence”;
What are all the delimiters you are considering?